File: CSharp/1047 - Tempo de Jogo com Minutos.cs
using System;

class MainClass {
  public static void Main (string[] args) {
    string[] S = Console.ReadLine().Split();
    int A = Convert.ToInt32(S[0]) * 60 + Convert.ToInt32(S[1]);
    int B = Convert.ToInt32(S[2]) * 60 + Convert.ToInt32(S[3]);
    int C = B - A;
    if (A >= B)
      C = 1440 - A + B;
    Console.WriteLine("O JOGO DUROU " + (C / 60) + " HORA(S) E " + (C % 60) + " MINUTO(S)"); 
  }
}

File: Python3/1047 - Tempo de Jogo com Minutos.py
A, B, C, D = map(int, input().split())
x = A * 60 + B
y = C * 60 + D
if A <= C:
  z = y - x
  if z == 0:
    print('O JOGO DUROU {:.0f} HORA(S) E {:.0f} MINUTO(S)'.format(24, z % 60))
  else:
    print('O JOGO DUROU {:.0f} HORA(S) E {:.0f} MINUTO(S)'.format((z - z % 60) / 60, z % 60))
else:
  z = (24 * 60 - x) + y
  print('O JOGO DUROU {:.0f} HORA(S) E {:.0f} MINUTO(S)'.format((z - z % 60) / 60, z % 60))
